src/components/SelectInput/SelectInput.tsx

```diff
@@ -188,7 +173,82 @@ const SelectInput = ({
   );
 };
 
-SelectInput.propTypes = propTypes;
-SelectInput.defaultProps = defaultProps;
+SelectInput.propTypes = {
```

Comment:
I think we should remove the comments/descriptions from the PropTypes. This will make it more clear where these are being sourced from to render the storybook docs.

Also, we decided a while back that we want to define the props before the function. So in the days of TS, I believe a component should generally look like:

```
interface Props {
 /**
    * Description
    */
  someProp?: boolean,
}

const propTypes = {
  someProp: PropTypes.string
};

const SomeComponent: FC<Props> = ({
  someProp = false
}) => {
...
};

SomeComponent.propTypes = propTypes;

export default SomeComponent;
```

Reply:
oh good point, i'll make both changes